examples/ipynb/webgl_example_1.ipynb

```diff
@@ -189,21 +191,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
```

Comment:
@bollu don't sweat it too much. FYI I usually try to do it when opening a PR, and before asking for final review. I inevitably find things I meant to change or didn't mean to change. It generally saves some time.